src/driver.ml

```diff
@@ -695,7 +695,8 @@ let map_structure_gen st ~tool_name ~hook ~expect_mismatch_handler ~input_name
       ~expect_mismatch_handler ~input_name ~embed_errors
   with
   | st, lint_errors, errors ->
```

Comment:
I cannot locate the comment on the whole relevant part.

Note that the following:
```
match expr
with pattern -> expr2
```
with only one pattern is equivalent to
```
let pattern = expr in
expr2
```
but the latter is more explicit and easier to read!

In this case, there were two cases (`Error` and `Ok`) and there is now only one, so it would make sense to turn the `match` into a `let` as explained above.